src/fmt/pl.c

```diff
@@ -6,7 +6,7 @@
 #include <ctype.h>
 #include <sys/types.h>
 #ifdef HAVE_STRINGS_H
-#define __EXTENSIONS__ 1
+#define _XPG4_2
```

Comment:
If `__EXTENSIONS__` defined, `strncasecmp()` is hidden on Illumos.  See their [strings.h](https://github.com/illumos/illumos-gate/blob/f7877f5d39900cfd8b20dd673e5ccc1ef7cc7447/usr/src/head/strings.h).